oeps/processes/oep-0055/decisions/0005-managing-core-contributors.rst

```diff
@@ -0,0 +1,71 @@
+#############################################################
+
+Status
+******
+
+**Under Review**
+
+Context
+*******
+
+Per `OEP-55
+<https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0055-proc-project-maintainers.html>`_,
+repository maintainers are responsible "for the health of the component and the
+component's community". Further, OEP-55 specifies that maintainers must
+participate in the "selection and approval of Core Contributors with commit
+rights to their repositories, by participating in the comment period for Core
+Contributor nominees nominated to commit code to their repo(s)".
+
+However, OEP-55 does not provide sufficient guidance around how Maintainers and
+Core Contributors should interact.
+
+Decision
+********
+
+Maintainers must participate in, and support, the Core Contributor program.
+Maintainers may not take actions that effectively prevent a Core Contributor
+from being able to carry out their Rights, as defined by `OEP-54
+<https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0054-core-contributors.html>`_.
+
+Maintainers should communicate architectural principles and code standards to
+Core Contributors on their repository. They may initiate the process to remove
+Core Contributors that do not follow standards or respond to feedback.
+
+Specification
+*************
+
+Maintainer Responsibilities
+===========================
+
+* Maintainers must participate in the selection of Core Contributors to their
+  repository by expressing their opinion of nominees during the `Core
+  Contributor nomination process <https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0054-core-contributors.html#new-core-contributor-nomination-process>`_
+
+* Maintainers must carefully weigh the merits of a nominee. Maintainers are not
+  allowed to object to every nominee in order to ensure they are the only people
+  with commit access on the repo. For nominees that are objected to, Maintainers
+  must provide feedback that includes what the nominee needs to do in order to
+  become a Core Contributor on the repository.
+
+* Maintainers may not take any action that would prevent existing Core
+  Contributors from being able to merge code. This includes, but is not limited
+  to, setting up branch protection rules that require a Maintainer to approve a
+  pull request before it can be merged.
+
+Maintainer Rights
+=================
+
+* Maintainers have the right to know who has Commit access on their repository,
+  and communicate their code standards and architectural principles with them.
+  This especially applies to new Maintainers taking over maintenance of an
+  existing repository. See the `maintainers how-to
+  <https://docs.openedx.org/en/latest/developers/how-tos/maintain-a-repo.html#your-ccs-are-aligned-with-your-expectations-as-maintainer>`_
+  for instructions on how to do this.
+
+* Maintainers have the right to remove Core Contributors who are not
```

Comment:
What is the process to remove a core contricutor?  It feels to me like giving maintainers the right to straight-up remove a contributor, without explicit recourse, might be too much.

Reply:
This may require further clarity, my reading was that maintainers and CCs have already spoken directly about the reason for the removal and have agreed.  If the removal is for misconduct, then the maintainer would announce the removal along with misconduct citations.   The removal which would come in as an axim request would either have a link to the public discussion/citations or would be confirmed with both the CC and maintainer before we proceed.

Reply:
Some of this is probably worth stating here.

Reply:
I meant they could initiate the CC removal process as described in https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0054-core-contributors.html#removing-core-contributors - I will clarify this.

Reply:
Wait... I do say this. 

> Maintainers have the right to remove Core Contributors who are not appropriately following Open edX community standards, Core Contributor standards (the "3 Cs") or project expectations, and have also not been responsive to Maintainer feedback. OEP-54 specifies how to initiate this process

Could you elaborate on what's unclear here? I feel this paragraph very clearly (to me) describes what I meant: Maintainers should provide feedback to CCs, and if that is unheeded, they can follow OEP-54's provision to begin the removal procedure. No where do I say this is a unilateral privilege that maintainers can just boot a CC.

Reply:
No need for further clarification.  I must have glossed over the details at the end of the paragraph.  Thanks, and sorry for the noise!